Library/Formula/bork.rb

```diff
@@ -0,0 +1,18 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.1.tar.gz"
+  sha256 "01ff60779582d5ec67080d43a6325d3f5f213dd3dbbbd1ba0989e0253bdc73aa"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match "checking: directory #{testpath}/foo\r" \
```

Comment:
OK, you can leave off the newlines, since those are included literally in the heredoc.  Why is the first line ended with `\r` rather than `\n`, though?